### ISSUE TITLE
add support output to markdown

### ISSUE THREAD

**Issue by gengjiawen** [yamadashy/repopack#86]

**Comment by yamadashy** [comment]
Thank you for your suggestion!
I had actually considered this before but it slipped my mind. 

For code blocks, we're planning to use triple backticks. There's a question of whether to escape backticks within the code itself, but I'll examine different patterns to determine the best approach.

I appreciate you bringing this up!

**Comment by yamadashy** [comment]
Hi, @gengjiawen 

I'm pleased to inform you that we've implemented the Markdown output support you suggested. This feature is now available in version 0.1.38 of Repopack, which has just been released!
https://github.com/yamadashy/repopack/releases/tag/v0.1.38

Thank you again for your valuable input. If you have a chance to try out the new Markdown output, we'd love to hear your thoughts!

**Comment by gengjiawen** [comment]
thx for the feature . But I notice two bugs. 

* output file extension still is txt
* no source type around ```

**Comment by yamadashy** [comment]
Thank you for your feedback and for pointing out these issues!
I'm pleased to inform you that both of these bugs have been addressed in my latest release, version 0.1.40:

https://github.com/yamadashy/repopack/releases/tag/v0.1.40

1. The output file extension now changes based on the selected style (e.g., .md for Markdown, .xml for XML).
2. I've added language identifiers to code blocks in Markdown output for better syntax highlighting.
